testgrid/cmd/configurator/config_test.go

```diff
@@ -190,10 +195,17 @@ func TestConfig(t *testing.T) {
 			}
 
 			// All dashboardtab within a dashboard must not have duplicated names
-			if dashboardtabmap[dashboardtab.Name] {
-				t.Errorf("Duplicated dashboardtab: %v", dashboardtab.Name)
+			if dashboardtabSet.Has(dashboardtab.Name) {
+				t.Errorf("Duplicated name in dashboard %s: %v", dashboard.Name, dashboardtab.Name)
+			} else {
```

Comment:
Do you need the else clause here?

Reply:
https://golang.org/pkg/testing/#B.Errorf

errorf marks the test as fail, but continues execution :-)